test/local/routes/password.js

```diff
@@ -45,15 +45,9 @@ function makeRoutes(options = {}) {
 }
 
 function runRoute(routes, name, request) {
-  return new P((resolve, reject) => {
-    getRoute(routes, name).handler(request, res => {
-      if (res instanceof Error) {
-        reject(res)
-      } else {
-        resolve(res)
-      }
-    })
-  })
+  return getRoute(routes, name).handler(request)
+  .then((res) => { return res })
```

Comment:
I think you can remove the `.then` and `.catch` lines since you are simply returning the value.